packages/cozy-stack-client/src/memoize.js

```diff
@@ -29,6 +29,26 @@ const memoize = (fn, options) => {
         result,
         date: Date.now()
       }
+      /**
+       * If the result is a promise and that this promise
+       * failed or resolved with a specific key (aka ''), let's remove
+       * the result from the cache since we don't want to
+       * memoize error
+       */
+
+      if (typeof result === 'object') {
+        if (typeof result.then === 'function') {
+          result
+            .then(v => {
+              if (v === '') {
```

Comment:
I think we should make this configurable, an empty string could be a valid value to memoize.

I wonder why in your case the called function returns '' as an error ?

Reply:
options.shouldCache = value => value !== ''

Reply:
What ares the errors that may return '' and nod false or undefined?

I agree with @ptbrowne and would prefer having the `if (v === '')` part to be a function given as option. Maybe `option.isError = v => (…)`

Reply:
> I wonder why in your case the called function returns '' as an error ?

I think this is because the return is used directly for <img src=$return />. I didn't want to change this current behavior. 

> What ares the errors that may return '' and nod false or undefined?

Typically, NO_INTERNET_CONNEXION, we currently return '' and not the error.